blocks/library/audio/index.js

```diff
@@ -105,22 +105,20 @@ registerBlockType( 'core/audio', {
 							<Dashicon icon="edit" />
 						</Button>
 					</Toolbar>
-				</BlockControls>
-			);
+				</BlockControls>,
 
-			const inspectorControls = focus && (
 				<InspectorControls key="inspector">
 					<BlockDescription>
 						<p>{ __( 'The Audio block allows you to embed audio files and play them back using a simple player.' ) }</p>
 					</BlockDescription>
 				</InspectorControls>
```

Comment:
I think you have a linting issue, a missing comma at the of this line. I'd recommend installing an "eslint" plugin for your favorite IDE/editor to see these errors inline. 

Alternatively you can run `npm run lint` locally to